spec/services/hyrax/resource_characterizer_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe Hyrax::ResourceCharacterizer do
+  subject(:characterizer) { described_class.new(source: resource) }
+  let(:resource) { FactoryBot.create(:file_set).valkyrie_resource }
```

Comment:
I think this is dead code since each context below has its own resource let block